chart/templates/clusterrole.yaml

```diff
@@ -0,0 +1,26 @@
+{{- if .Values.rbac.create }}
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "fullname" . }}
+    {{- if (.Values.teamName) }}
+    team: {{ .Values.teamName }}
+    {{- end}}
```

Comment:
Would it make sense to add a field _extraLabels_ in the `values.yaml` to allow the user to add extra labels other than team?
Most of the charts do this with annotations.

Reply:
Example so I can add this without thinking? ;)

Reply:
`values.yaml`:
```
extraLabels: {}  
  # team: core
```

templates:
```
  labels:
    app: {{ template "fullname" . }}
    {{- if (.Values.extraLabels) }}
{{ toYaml .Values.extraLabels| indent 4 }}
    {{- end}}
```